.maintain/common/lib.sh

```diff
@@ -101,3 +101,18 @@ structure_message() {
 send_message() {
 curl -XPOST -d "$1" "https://matrix.parity.io/_matrix/client/r0/rooms/$2/send/m.room.message?access_token=$3"
 }
+
+# Check for runtime changes between two commits. This is defined as any changes
+# to bin/node/src/runtime, frame/ and primitives/sr_* trees.
```

Comment:
`bin/node/runtime` should not be relevant for anything here. `primitives/sr_*` does not exist anymore

Reply:
Ah so in that case, check_runtime.sh has been a bit broken in substrate for some time... Are there any dirs in `primitives/` that we'd want to explicitly exclude?

Reply:
I think we more want an explicit inclusion for crates in `primitives` like `arithmetic`

Reply:
I actually think arithmetic should be included, and Ideally I want to give the whole package to the audit team at some point. 

Agree that `bin/node/runtime` is not important. 

But I want to point out that this will run into issues as Ideally we want to only monitor code that reaches polkadot, things like assets, example, or gilt also live in `pallet` but are not in polkadot and need not be audited.

Reply:
So that's fine - the changes can be labelled as trivial. Barring some sort of 'intelligent' solution that could determine weather a change in Substrate impacts Polkadot, I can't really think of a better way to do it.

Ok to finish this PR up, shall we either include *everything* in /primitives, or have a list of subdirectories we check? If the latter, I need a list of which subdirs to check